src/crate/sphinx/csv.py

```diff
@@ -7,20 +7,48 @@
 from docutils.parsers.rst.directives.tables import CSVTable
 from docutils.utils import SystemMessagePropagation
 from docutils import statemachine
-from docutils.parsers.rst import directives
+
+def exclude_dict(argument):
+    return ast.literal_eval(argument)
+
+def non_negative_int(argument):
+    """
+    Converts the argument into an integer.
+    Raises ValueError for negative or non-integer values.
+    """
+    value = int(argument)
+    if value >= 0:
+        return value
+    else:
+        raise ValueError('negative or non-integer value; must be non-negative')
```

Comment:
this `ValueError` can only be raised if `argument` was already a convertable integer